web_hr_recruitment_label/models/hr_job_label_category.py

```diff
@@ -0,0 +1,14 @@
+from odoo import fields, models
+
+
+class HrJobLabelCategory(models.Model):
+    _inherit = "hr.job.label.category"
+
+    allow_filtering = fields.Boolean(
+        default=True,
+        help="Disable showing this category on the filter options on the website",
+    )
+    show_on_website = fields.Boolean(
```

Comment:
Not sure, but is it possible to use `website.published.mixin` for this? or has it a diff functionality

Reply:
I think the mixin has more features than needed, it just needs the boolean, not any of the checks or an url.